opensrp-chw/src/main/java/org/smartregister/chw/interactor/ChildProfileInteractor.java

```diff
@@ -46,6 +50,7 @@
     public static final String TAG = ChildProfileInteractor.class.getName();
     private AppExecutors appExecutors;
     private Map<String, Date> vaccineList = new LinkedHashMap<>();
+    ChildProfileInteractor.Flavour childProfileInteractorFlv = new ChildProfileInteractorFlv();
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5708017)